client/state/sites/plans/reducer.js

```diff
@@ -34,6 +35,9 @@ export function plans( state = {}, action ) {
 			} );
 		case REMOVE_SITE_PLANS:
 			return omit( state, action.siteId );
+		case SERIALIZE:
+			//TODO: we have full instances of moment.js on sites.plans[siteID].data
```

Comment:
:smile: feel free to grab https://github.com/Automattic/wp-calypso/issues/2757 but this currently isn't blocking me.